Source/WebCore/platform/graphics/ColorMatrix.h

```diff
@@ -72,8 +72,33 @@ constexpr bool operator!=(const ColorMatrix<ColumnCount, RowCount>& a, const Col
     return !(a == b);
 }
 
+constexpr ColorMatrix<3, 3> brightnessColorMatrix(float amount)
+{
+    // Brightness is specified as a compontent transfer function: https://www.w3.org/TR/filter-effects-1/#brightnessEquivalent
+    // which is equivalent to the following matrix.
+    amount = std::max(amount, 0.0f);
+    return ColorMatrix<3, 3> {
+        amount, 0.f, 0.f,
```

Comment:
I have been trying to use `0.0f` consistently rather than leaving off the trailing `0`. (same comment for below).